autoname_workspaces.py

```diff
@@ -136,5 +144,9 @@ def window_event_handler(i3, e):
         if e.change in ['new', 'close', 'move']:
             rename_workspaces(i3)
 
+    def workspace_event_handler(i3, e):
+        rename_workspaces(i3)
```

Comment:
I think we only want to trigger a rename when a workspace is created or deleted. Similar to the `window_event_handler()`, we should check the value of `e.change`.

See the docs here for what values "change" could be https://i3wm.org/docs/ipc.html#_events.